prdoc/pr_7026.prdoc

```diff
@@ -0,0 +1,17 @@
+# Schema: Polkadot SDK PRDoc Schema (prdoc) v1.0.0
+# See doc at https://raw.githubusercontent.com/paritytech/polkadot-sdk/master/prdoc/schema_user.json
+
+title: [pallet-broker] add extrinsic to remove a lease
```

Comment:
```suggestion
title: "[pallet-broker] add extrinsic to remove a lease"
```
I think the unquoted square brackets are the reason why the prdoc and semver checks are failing